doc: include library artifacts (#179)
ansys/pyworkbench: doc/source/artifacts.rst

Artifacts
#########

> This page allows to download the different artifacts generated by the build

Reviewer: ```suggestion
This page lets you download the different artifacts generated by the build
```
